lib/code_fund_web/controllers/track_controller.ex

```diff
@@ -8,7 +8,7 @@ defmodule CodeFundWeb.TrackController do
   def pixel(conn, %{"property_id" => property_id} = params) do
     try do
       property = Properties.get_property!(property_id)
-      sponsorship = Sponsorships.get_sponsorship_for_property(property)
+      sponsorship = Sponsorships.get_and_update_sponsorship_for_property(property)
```

Comment:
I'm thinking we need to deprecate this controller action. The only pixels we should be tracking should include the sponsorship ID and not the property id

Reply:
I think this makes a lot of sense, especially given the suggested schema refactor, as you should be able to discern the property that was acted upon. Semantically, I wonder if this isn't a responsibility of the `ImpressionsController`.

Reply:
Agreed. Feel free to refactor as you deem appropriate